engine/flink/components/base/src/test/scala/pl/touk/nussknacker/engine/flink/util/transformer/aggregate/TransformersTest.scala

```diff
@@ -151,6 +154,43 @@ class TransformersTest extends FunSuite with FlinkSpec with Matchers with Inside
     aggregateVariables shouldBe List(3, 5)
   }
 
+  test("sum tumbling aggregate when in fragment") {
```

Comment:
Can we add a test for validation that shows that e.g. input is not accessible after fragment?